app/controllers/application_controller.rb

```diff
@@ -2,4 +2,9 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user
+
+  def current_user
+    @user ||= User.find_by(id: session[:user_id])
```

Comment:
You can shorten this a little bit (like you did in the suggestions controller!) by just using the `find` method, which will automatically find by id.